src/lib.rs

```diff
@@ -247,6 +268,14 @@ pub fn metadata_deps(manifest_path: Option<&Path>, deps: bool) -> Result<Metadat
         cmd.arg("--no-deps");
     }
 
+    if features.is_some() {
```

Comment:
`if let Some(features) = features` saves you the unwrap

Reply:
done, thanks for the feedback, is a rust-beginner contribution ;)